### PR TITLE
Np 1792 uri as identifier

### PR DESCRIPTION
**Pull request by svenngjeto**
Quickfix to allow change of registrered URIs

### CODE REVIEW

File: src/test/java/no/unit/nva/bare/UpdateAuthorityIdentifierHandlerTest.java [hunk 1]
```diff
@@ -221,6 +230,39 @@ public void handlerReturnsOkWhenInputIsValidAndAuthorityIdentifierIsUpdatedSucce
...
     }
 
+    @Test
+    @DisplayName("handler Returns Ok Response When Input Is Valid URI And Authority Identifier Is Updated Successfully")
+    public void handlerReturnsOkWhenInputIsValidUriAndAuthorityIdentifierIsUpdatedSuccessfully() throws Exception {
+
+        InputStream is =
+                UpdateAuthorityIdentifierHandler.class.getResourceAsStream(BARE_SINGLE_AUTHORITY_GET_RESPONSE_JSON);
+        final BareAuthority bareAuthority = new Gson().fromJson(new InputStreamReader(is), BareAuthority.class);
+        Config.getInstance().setBareHost("localhost");
```
Comment (brinxmat) [inline src/test/java/no/unit/nva/bare/UpdateAuthorityIdentifierHandlerTest.java:240]: constant

File: src/test/java/no/unit/nva/bare/UpdateAuthorityIdentifierHandlerTest.java [hunk 2]
```diff
@@ -221,6 +230,39 @@ public void handlerReturnsOkWhenInputIsValidAndAuthorityIdentifierIsUpdatedSucce
...
+        HttpClient httpClient = mock(HttpClient.class);
```
Comment (brinxmat) [inline src/test/java/no/unit/nva/bare/UpdateAuthorityIdentifierHandlerTest.java:241]: can you extract the mock creation to a suitably named method?

**Review by brinxmat — CHANGES_REQUESTED** [review]

File: src/test/java/no/unit/nva/bare/UpdateAuthorityIdentifierHandlerTest.java (hunk 1 above)
Comment (svenngjeto) [reply src/test/java/no/unit/nva/bare/UpdateAuthorityIdentifierHandlerTest.java]: Moved to constant

File: src/test/java/no/unit/nva/bare/UpdateAuthorityIdentifierHandlerTest.java (hunk 2 above)
Comment (svenngjeto) [reply src/test/java/no/unit/nva/bare/UpdateAuthorityIdentifierHandlerTest.java]: Extracted two methods, one for bareConnection and one for updateHandler

**Review by brinxmat — APPROVED** [review]

**Pull request merged**